example/src/App.tsx

```diff
@@ -84,6 +86,11 @@ export default function App() {
     console.log('in verify');
     if (adaptor.localStream.current && adaptor.localStream.current.toURL()) {
       console.log('in verify if adaptor local stream', adaptor.localStream);
+      if (isWaitingWebsocketInit) {
+        setIsWaitingWebsocketInit(false);
+        publishStreamId = generateRandomString(12);
```

Comment:
What is the idea of this line (91)? Is it possible to remove this line? Without it it works as expected !

Reply:
the idea is preventing the stream id already in use issue but if you set it according to your use case, you can use the same stream if all the time.